specification/network/resource-manager/Microsoft.Network/preview/2024-03-01/webapplicationfirewall.json

```diff
@@ -835,6 +843,25 @@
         }
       }
     },
+    "ManagedRuleSetRuleGroup": {
+      "description": "Defines a managed rule set rule group",
+      "required": [
+        "ruleGroupName"
+      ],
+      "properties": {
+        "ruleGroupName": {
+          "description": "Name of the rule group",
+          "type": "string"
+        },
+        "rules": {
+          "description": "List of rules within the rule group",
```

Comment:
Wondering if there are any constraints applicable for rules or customer can give any string ?